apps/roam/src/components/settings/NodeConfig.tsx

```diff
@@ -229,7 +229,7 @@ const NodeConfig = ({
                 onChange={handleTagChange}
                 onBlur={handleTagBlur}
                 error={tagError}
-                placeholder={`#${node.text}`}
```

Comment:
Actually I liked the `#` 😁.

I'm going to create a task to handle `#`, as we discussed today.  Re can re-add it once the use case is handled 😃

Reply:
[ENG-693: Handle `#` in tags](https://linear.app/discourse-graphs/issue/ENG-693/handle-in-tags)